opentech/urls.py

```diff
@@ -14,8 +16,13 @@
 from opentech.apply.users.urls import public_urlpatterns as user_urls
 
 
+def redirect_to_my_auth(request):
+    return redirect_to_login(reverse('wagtailadmin_home'), login_url='users_public:login')
+
+
 urlpatterns = [
     path('django-admin/', admin.site.urls),
+    path('admin/login/', redirect_to_my_auth, name='wagtailadmin_login'),
```

Comment:
We can directly call the LoginView here instead of creating a new view. Also, using reverse('wagtailadmin_home') user will be redirected to Wagtail home page after login. But we should redirect the user to the requested page. Using view directly will help the request for `/admin/pages/1/` to `/admin/login/?next=/admin/pages/1/` instead of `/login/?next=/admin/`.

```python
path('admin/login/', LoginView.as_view(template_name='users/login.html'), name='wagtailadmin_login'),
```

Reply:
A lot neater, fixed now.